blog/_posts/2022-09-06-ads-object-search.md

```diff
@@ -0,0 +1,91 @@
+
+From a technical point of view, whether the aforementioned query finds more than just records that literally mention `Aldebaran` in their basic metadata, depends on if the search engine has been made aware of the fact that the star Aldebaran is also known in other ways. These are the so-called synonyms. Whether the ADS has synonyms listed for Aldebaran (or any term, for that matter) can be checked by running a query for the term and excluding any literal mentions of that term (by switching off synonym replacement, using the = modifier)
+
+```
+Aldebaran NOT =Aldebaran collection:astronomy
+```
+
+In our [example](https://ui.adsabs.harvard.edu/search/q=Aldebaran%20NOT%20%3DAldebaran%20collection%3Aastronomy&sort=date%20desc%2C%20bibcode%20desc&p_=0), this just returns 1 record. The `Show Highlights` feature in the search results page shows that the ADS has the synonym Aldeberan included in its index. This means that the ADS is NOT aware of synonyms like: Alpha Tauri, alf Tau, alpha Tau, HD 29139, GC 5605. How does this impact search results? For example, the following query returns about 200 more records:
```

Comment:
It took me a sec - maybe add this to this sentence: `The Show Highlights feature in the search results page shows that the ADS has the synonym Aldeberan (note the misspelling) included in its index.`